gallery/src/components/NxButton/NxButtonPage.tsx

```diff
@@ -126,7 +126,8 @@ export default function NxButtonPage() {
                           id="nx-button-icon-only-example"
                           liveExample={NxButtonIconOnlyExample}
                           codeExamples={nxButtonIconOnlyCode}>
-        An example of a button containing only an icon. For accessibility purposes it is important to use the
+        An example of buttons containing only an icon and with one that is in a disabled state.
```

Comment:
This sentence is no longer accurate.  Also I think it would make sense to explain here when you would use each of the different ways of disabling, and that the `title` is not required (and doesn't really work right) on attr-disabled icon buttons

Reply:
> Thanks for pointing that out!
I've updated it here, let me know what you think: a7465be

The last sentence could be confusing as-is.  Maybe explain that the title is important on all enabled buttons but that it may or may not be on a disabled button depending on the situation.